node_helper.js

```diff
@@ -1,6 +1,10 @@
-const fs = require("fs");
-const path = require("path");
-const validUrl = require("valid-url");
+const fs = require("fs")
+const path = require("path")
+const validUrl = require("valid-url")
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```

Comment:
fetch, moment, ICAL, IcalExpander now are two times here.

base64 we can be removed.

Reply:
Yes, was a merge problem. If fixed now!